app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,57 @@
 @import "base/base";
 
 @import "layout";
+
+// variables
+$alert: #de1b31;
+$error: #de1b31;
+$notice: #000;
+$success: #2adc55;
+
+.nav {
+
+    .menu {
+        @include outer-container(68em);
+        list-style-type: none;
+
+        .menu-link {
+            @include span-columns(2);
+
+            font-size: 1.3em;
+            font-weight: bold;
+            margin: 0.5em 0;    
+        }   
+    }
+
+    border-bottom: solid 1px #ddd;
+    box-shadow: 0px 0px 1px #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>`0px` should be written without units as `0`